# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/FileManager.h
# Commit: fddd01727ad2
# Full Hash: fddd01727ad2f5c097b4d750bba958d0cdfc1340
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-05-14 16:15:31
# Regressor Bug: 1623278
# File Overlap Count: 1
# Description:
#   Bug 1623278 - Use MovingNotNull in ActorsParent. r=dom-workers-and-storage-reviewers,janv
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D73033
# ==============================================================================

diff -r 5f5964262d18 -r fddd01727ad2 dom/indexedDB/FileManager.h
--- a/dom/indexedDB/FileManager.h	Thu May 14 09:14:04 2020 +0000
+++ b/dom/indexedDB/FileManager.h	Thu May 14 09:14:17 2020 +0000
@@ -68,7 +68,7 @@
 
   bool EnforcingQuota() const { return mEnforcingQuota; }
 
-  nsresult Init(nsIFile* aDirectory, mozIStorageConnection* aConnection);
+  nsresult Init(nsIFile* aDirectory, mozIStorageConnection& aConnection);
 
   [[nodiscard]] nsCOMPtr<nsIFile> GetDirectory();
 
